helix-core/src/main/java/org/apache/helix/monitoring/metrics/implementation/RebalanceLatencyGauge.java

```diff
@@ -48,7 +48,6 @@ public RebalanceLatencyGauge(String metricName, long slidingTimeWindow) {
    * the rebalancer could fail at any point, and we want it to recover gracefully by resetting the
    * internal state of this metric.
    */
-  @Override
```

Comment:
Why was this removed?

Reply:
Strictly speaking, the implementation of the interface method is not overridden. But as I searched its java doc, it will help the compiler to validate. So let me add it back.